recipe/build.sh

```diff
@@ -3,11 +3,21 @@ export PATH="$PREFIX/bin:$BUILD_PREFIX/bin:$PATH"
 #export LDFLAGS="-Wl,-L$PREFIX/lib $LDFLAGS"
 export LD_LIBRARY_PATH="$PREFIX/lib:$LD_LIBRARY_PATH"
 export LIBRARY_PATH="$PREFIX/lib:$LIBRARY_PATH"
+if [ $ARCH == "aarch64" ]; then
+   echo "aarch64 detected"
+fi
 ./configure --prefix=$PREFIX --with-gmp-includes=$PREFIX/include --with-gmp-libraries=$PREFIX/lib
 make install
-#Small test
-if [ -f "$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-ld" ]; then
-   ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-ld $BUILD_PREFIX/bin/ld
+# make sure this is more arch specific
+if [ $ARCH == "x86_64" ]; then
+  if [ -f "$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-ld" ]; then
+     ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-ld $BUILD_PREFIX/bin/ld
+  fi
+fi
+if [ $ARCH == "aarch64" ]; then
+  if [ -f "$BUILD_PREFIX/bin/aarch64-conda_cos6-linux-gnu-ld" ]; then
+     ln -s $BUILD_PREFIX/bin/aarch64-conda_cos6-linux-gnu-ld $BUILD_PREFIX/bin/ld
+  fi
 fi
```

Comment:
Haven't tested, but something like below should work.
```suggestion
if [[ -f "$LD" && ! $BUILD_PREFIX/bin/ld ]]; then
   ln -s $LD $BUILD_PREFIX/bin/ld;
fi
```

Reply:
yeah, maybe. tbh this was also there already from the original recipe, so not sure how badly it is needed.